.travis.yml

```diff
@@ -15,6 +15,8 @@
 
 language: java
 
+script: mvn clean verify -Dlicense.skip=false
```

Comment:
Do we need to run verify? By default travis-ci uses "mvn test". Should it be "mvn license:check test -Dlicense.skip=false"?

Are the following warnings expected?

```
[WARNING] Unknown file extension: /Users/vrozov/Projects/DataTorrent/apex/NOTICE
[WARNING] Unknown file extension: /Users/vrozov/Projects/DataTorrent/apex/LICENSE
[WARNING] Unknown file extension: /Users/vrozov/Projects/DataTorrent/apex/DISCLAIMER
[WARNING] Unable to find a comment style definition for some files. You may want to add a custom mapping for the relevant file extensions.
```